marklogic-junit5/examples/simple-dhf4/build.gradle

```diff
@@ -10,9 +10,9 @@ repositories {
 }
 
 dependencies {
-  mlRestApi "com.marklogic:marklogic-unit-test-modules:0.13.develop"
+  mlRestApi "com.marklogic:marklogic-unit-test-modules:1.0.beta"
 
-	testCompile "com.marklogic:marklogic-junit:0.13.develop"
+	testCompile "com.marklogic:marklogic-junit5:1.0.beta"
```

Comment:
Do not publish with a dependency on a beta release. Need to update junit5 and marklogic-unit-test-modules dependencies

Reply:
Fixed in pr #101